internal/gatewayapi/testdata/httproute-with-single-rule-with-multiple-rules.out.yaml

```diff
@@ -92,11 +92,16 @@ httpRoutes:
     parents:
     - conditions:
       - lastTransitionTime: null
-        message: 'Regex "*regex*" is invalid: error parsing regexp: missing argument
-          to repetition operator: `*`.'
+        message: 'Failed to process route rule 2: regex "*regex*" is invalid: error
+          parsing regexp: missing argument to repetition operator: `*`.'
         reason: UnsupportedValue
         status: "False"
         type: Accepted
+      - lastTransitionTime: null
+        message: Resolved all the Object references for the Route
+        reason: ResolvedRefs
+        status: "True"
+        type: ResolvedRefs
```

Comment:
The behavior follows the existing logic: Even though the `Accepted` condition is false, the `ResolveRefs` can be true if all the Backends are successfully resolved.

Existing tests:

https://github.com/envoyproxy/gateway/blob/652b6b4c5141cbeb227eebc6ca2358c108942079/internal/gatewayapi/testdata/gateway-with-listener-with-invalid-tls-configuration-secret-in-other-namespace.out.yaml#L65-L76